fiat-web/src/main/java/com/netflix/spinnaker/fiat/controllers/AuthorizeController.java

```diff
@@ -191,4 +193,31 @@ public void getUserAuthorization(@PathVariable String userId,
 
     response.setStatus(HttpServletResponse.SC_NOT_FOUND);
   }
+
+  private Optional<UserPermission> getUserPermissionOrDefault(String userId) {
```

Comment:
Does this seem reasonable?

Aiming to support the use case of one of us needing to talk to an underlying service directly (ie. `clouddriver`) and not go through `gate`.

Reply:
Have you tried adding `-H X-SPINNAKER-USER:username@org.com` to your curl command?

Reply:
This seems like a good starting point, we may eventually tease out some use cases around ops or propagating admin level credentials out of this, but I'm good with starting here

Reply:
That's a good point ... I did not set the header as I was coming in through the browser.